assets/src/components/inputModal.tsx

```diff
@@ -9,22 +10,18 @@ const InputModal = ({
 }) => {
   const [, dispatch] = useContext(StateDispatchContext)
   return (
-    <>
-      <div role="dialog">
-        <div
-          className="c-input-modal"
-          role="presentation"
-          onKeyDown={(event) => {
-            if (event.key === "Escape") {
-              dispatch(closeInputModal())
-            }
-          }}
-        >
-          {children}
-        </div>
-      </div>
-      <div className="c-input-modal-backdrop" aria-hidden={true} />
-    </>
+    <Modal
+      className="c-input-modal"
+      onHide={() => {
+        dispatch(closeInputModal())
+      }}
+      show
+      renderBackdrop={(props) => (
+        <div {...props} className="c-modal-backdrop" aria-hidden />
```

Comment:
This may be a silly question, but the `className` changed from `c-input-modal-backdrop` to `c-modal-backdrop`. Does that make a difference?